pxr/base/plug/wrapRegistry.cpp

```diff
@@ -54,6 +50,23 @@ PXR_NAMESPACE_USING_DIRECTIVE
 
 namespace {
 
```

Comment:
I feel like this would be simpler if we just change the PluginNames function to no longer be templated and just take two `const PlugPluginPtrVector::iterator &` parameters instead for begin and end. This especially given it's only called in two places.

Reply:
Done!